src/components/EditableField/EditableField.jsx

```diff
@@ -956,7 +958,9 @@ export class EditableField extends React.Component {
         {fieldValue.map((val, index) => {
           const isActive = activeField === val.id
           const isDisabled =
-            val.disabled || this.state.disabledItem.indexOf(val.id) !== -1
+            disabled ||
+            val.disabled ||
+            this.state.disabledItem.indexOf(val.id) !== -1
```

Comment:
we could use includes instead of indexOf here

```suggestion
            this.state.disabledItem.includes(val.id)
```

Reply:
Cool, done!